### ISSUE TITLE
Mathjax display mode not working

### ISSUE THREAD

**Issue by Olafyii** [Huxpro/huxpro.github.io#378]
![image](https://user-images.githubusercontent.com/23097437/89611010-9a213c80-d8ae-11ea-9855-2b99dd1a3949.png)
->
![image](https://user-images.githubusercontent.com/23097437/89610856-3d258680-d8ae-11ea-9cb8-27d61825c3c8.png)

mathjax_support.html:
```
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    TeX: {
      equationNumbers: {
        autoNumber: "AMS"
      }
    },
    SVG: {
      scale: 90
    },
    tex2jax: {
      inlineMath: [ ['$','$'] ],
      displayMath: [ ['$$','$$'], ['[[', ']]'] ],
      processEscapes: true,
    }
  });
</script>
<script type="text/javascript"
        src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>
```
Why is the double dollar delimiter not working?

**Comment by Huxpro** [comment]
@Olafyii 

Did you turn on the Latex support via `mathjax: true` as mentioned in the <https://github.com/Huxpro/huxpro.github.io/blob/master/_doc/Manual.md#posts>?

**Comment by Olafyii** [comment]
@Huxpro 
Yes, and the inline mode is working fine. The display mode also works if the delimiter is not ['$$','$$'], for example ['[[',']]']. But if I use '\$$' as delimiter, it works.
![image](https://user-images.githubusercontent.com/23097437/89697950-326ffd80-d951-11ea-9a06-48932abbf4e2.png)
Line 1, 3, and 4 works just fine.
Seems that the '$$' delimiter won't invoke mathjax, and will change to '\(' in HTML...

**Comment by roife** [comment]
I cannot have `$$` work as well. But `$` works.

**Comment by jinyiliu** [comment]
我也有一样的问题，本地预览的时候，用 Chrome 是可以正常显示公式的，但是 Safari 显示有问题，所有 $$ 都变成 \] ，但是 $ 是没有问题的。然而 push 之后 Chrome 也不能正常显示了，Chrome 和 Safari 都只有 $ 能显示。最近才出现的问题，以前没遇到过。

**Comment by mikelyou** [comment]
Same problem here. My formulas that were good in the past now don't show correctly. `$xx$` still works, but `$$xxx$$` is broken. Maybe it's due to the version upgrading of mathjax, I guess?

**Comment by zwt0204** [comment]
> 我也有一样的问题，本地预览的时候，用 Chrome 是可以正常显示公式的，但是 Safari 显示有问题，所有 $$ 都变成 \] ，但是 $ 是没有问题的。然而 push 之后 Chrome 也不能正常显示了，Chrome 和 Safari 都只有 $ 能显示。最近才出现的问题，以前没遇到过。

参考https://github.com/fancyerii/fancyerii.github.io/blob/master/_includes/mathjax.html
已解决

**Comment by zwt0204** [comment]
@jinyiliu 
参考https://github.com/fancyerii/fancyerii.github.io/blob/master/_includes/mathjax.html
已解决

**Comment by jinyiliu** [comment]
@zwt0204 谢谢，问题解决了！

**Comment by simplestory** [comment]
我更换为下面这个，可以解决，各字符进行各种操作都显示正常

```
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [['$','$'], ['\\(','\\)']],
      processEscapes: true,
    }
  });
</script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML' async></script>
```

**Comment by selchwang** [comment]
`$xx$` and `$$xx$$` works well with this config
```html
// _includes/mathjax_support.html
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [['$','$'], ['\\(','\\)']],
      displayMath: [['$$','$$'], ['\\(','\\)']],
      processEscapes: true,
    }
  });
</script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML' async></script>
```

**Comment by Yingfan-Duan** [comment]
> 我更换为下面这个，可以解决，各字符进行各种操作都显示正常
> 
> ```
> <script type="text/x-mathjax-config">
>   MathJax.Hub.Config({
>     tex2jax: {
>       inlineMath: [['$','$'], ['\\(','\\)']],
>       processEscapes: true,
>     }
>   });
> </script>
> <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML' async></script>
> ```

This approach solved my problem as well! Thx
